ArcGISRuntimeSDKQt_QMLSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,374 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import Esri.ArcGISRuntime 100.13
+import Esri.ArcGISExtras 1.1
+
+
+Rectangle {
+    id: sampleWindow
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url dataPath: System.userHomePath + "/ArcGIS/Runtime/Data/"
+
+    // The X,Y coordinates from which to open the attribute pane and place new features
+    property int attributePromptX: 0
+    property int attributePromptY: 0
+
+    // Feature to define attributes for with ContingentValues
+    property var newFeature: null;
+
+    // TaskId to track the queryFeatures() function of the GeodatabaseFeatureTable
+    property var taskId;
+
+    // GeodatabaseFeatureTable
+    property var birdNestsTable: null;
+
+    // Field Domains are hardcoded to simplify this sample, but can be obtained from a FeatureTable's Domain
+    property var statusValues: {"Occupied":"OCCUPIED", "Unoccupied":"UNOCCUPIED"};
+    property var protectionValues: {"Endangered":"ENDANGERED", "Not endangered":"NOT_ENDANGERED", "N/A":"NA"};
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        MouseArea {
+            anchors.fill: mapView
+            visible: attributePrompt.visible
+            onClicked: mouse.accepted = attributePrompt.visible
+            onWheel: wheel.accepted = attributePrompt.visible
+        }
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+
+        Map {
+            Basemap {
+                ArcGISVectorTiledLayer {
+                    url: dataPath + "vtpk/FillmoreTopographicMap.vtpk"
+                }
+            }
+
+            ViewpointCenter {
+                Point {
+                    x: -13236000
+                    y: 4081200
+                    spatialReference: SpatialReference { wkid: 3857 }
+                }
+                targetScale: 8822
+            }
+
+            FeatureLayer {
+                id: nestsLayer
+                featureTable: birdNestsTable ?? null;
+
+                Geodatabase {
+                    id: gdb
+                    path: dataPath + "geodatabase/ContingentValuesBirdNests.geodatabase"
+
+                    onLoadStatusChanged: {
+                        if (loadStatus === Enums.LoadStatusLoaded) {
+                            birdNestsTable = gdb.geodatabaseFeatureTablesByTableName["BirdNests"];
+                            birdNestsTable.load();
+
+                            // Load the ContingentValuesDefinition to enable access to the GeodatabaseFeatureTable's ContingentValues data such as FieldGroups and ContingentValuesResults
+                            birdNestsTable.contingentValuesDefinition.load();
```

Comment:
Upon opening the sample I get the following error referring to this line. Removing the call to load() does not seem to affect the sample.

> qrc:/Samples/EditData/ContingentValues/ContingentValues.qml:95: TypeError: Cannot call method 'load' of undefined

Reply:
My sample will crash if I don't call `contingentValuesDefinition.load()` which is expected behavior, weird. Well I moved that call to lower down and it shouldn't trigger until `birdNestsTable` loads.